yb-voyager/src/srcdb/postgres.go

```diff
@@ -1136,27 +1136,27 @@ func (pg *PostgreSQL) GetMissingExportDataPermissions(exportType string, finalTa
 	return combinedResult, nil
 }
 
-func (pg *PostgreSQL) GetMissingAssessMigrationPermissions() ([]string, error) {
+func (pg *PostgreSQL) GetMissingAssessMigrationPermissions() ([]string, bool, error) {
 	var combinedResult []string
 
 	// Check if tables have SELECT permission
 	missingTables, err := pg.listTablesMissingSelectPermission("")
 	if err != nil {
-		return nil, fmt.Errorf("error checking table select permissions: %w", err)
+		return nil, true, fmt.Errorf("error checking table select permissions: %w", err)
```

Comment:
false, instead of true?

Reply:
in case of err i don't think that matters... since we haven't checked for pgss also at that point.
Will change it to false, just to be on safer side.

Reply:
Ah, right. I somehow thought even when error is returned, we would prompt the user and ask if we can continue, but that's not the case. 
Either way, yeah, to be on the safer side, false is better.